### PR TITLE
Features/warmup with documents

### PR DESCRIPTION
**Pull request by Raytlty**
1. remove eip712 signer because this is PersonSign kind sign.
2. mark `sha1` algorithm is not safe.[reason](https://sha-mbles.github.io/)
3. simple doc without doctest

### CODE REVIEW

File: rings-core/src/prelude.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//! prelude
+//! Re-exports of important traits, types, and functions used with rin-core.
```
Comment (googolmo) [inline rings-core/src/prelude.rs:1]: Typo rings–core

**Review by googolmo — CHANGES_REQUESTED** [review]

File: rings-core/src/ecc/signers.rs [hunk 2]
```diff
@@ -33,24 +36,82 @@ pub mod default {
...
+        if let Ok(p) = recover(msg, sig) {
+            p.address() == *address
+        } else {
+            false
+        }
+    }
+}
+
+/// PersonSign mod.
+pub mod person_sign {
```
Comment (RyanKung) [inline rings-core/src/ecc/signers.rs:92]: rename to EIP191

Comment (RyanKung) [reply rings-core/src/ecc/signers.rs:92]: ref: https://eips.ethereum.org/EIPS/eip-191

File: rings-core/src/ecc/signers.rs [hunk 3]
```diff
@@ -116,7 +180,7 @@ mod test {
     }
 
     #[test]
-    fn test_eip712_sign() {
```
Comment (RyanKung) [inline rings-core/src/ecc/signers.rs:119]: we should keep both `test_eip712_*`, and `test_eip191_*`

**Comment by RyanKung** [comment]
As I understand, the original EIP712 is EIP191, and the current person_sign method is actually EIP191.
So, should we support SIGN as EIP712? Or EIP191 is good enough?

File: rings-core/src/message/handlers/custom.rs [hunk 4]
```diff
@@ -16,6 +16,7 @@ use crate::transports::manager::TransportManager;
 #[cfg_attr(feature = "wasm", async_trait(?Send))]
 #[cfg_attr(not(feature = "wasm"), async_trait)]
 impl HandleMsg<MaybeEncrypted<CustomMessage>> for MessageHandler {
+    ///
```
Comment (RyanKung) [inline rings-core/src/message/handlers/custom.rs:19]: ？

File: rings-core/src/message/types.rs [hunk 5]
```diff
@@ -10,100 +10,124 @@ use crate::err::Error;
 use crate::err::Result;
 
 #[derive(Debug, PartialEq, Eq, Deserialize, Serialize, Clone)]
+/// MessageType use to ask for connection, send to remote with transport_uuid and handshake_info.
```
Comment (RyanKung) [inline rings-core/src/message/types.rs:13]: Coding style

```rust
/// Comment
#[derive]
pub struct foo {
}
```

**Review by RyanKung — APPROVED** [review]

**Review by googolmo — APPROVED** [review]

**Pull request merged**